ironfish/src/rpc/routes/chain/getNetworkInfo.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { ApiNamespace, router } from '../router'
+
+export type GetNetworkInfoRequest = undefined
+export type GetNetworkInfoResponse = {
+  networkId?: number
+}
+
+export const GetNetworkInfoRequestSchema: yup.MixedSchema<GetNetworkInfoRequest> = yup
+  .mixed()
+  .oneOf([undefined] as const)
+
+export const GetNetworkInfoResponseSchema: yup.ObjectSchema<GetNetworkInfoResponse> = yup
+  .object({
+    networkId: yup.number().optional(),
```

Comment:
Nice, this looks good to me. One more thing, could you see if it's possible to remove this `.optional()`? The network ID might be always initialized to the default value, but I'm not sure.